requirements/prod.txt

```diff
@@ -8,5 +8,7 @@ django-crispy-forms==1.4.0
 django-guardian==1.2.5
 django-imagekit==3.2.6
 djangocms-admin-style==0.2.5
+django_countries
```

Comment:
Check #123, #151 where use of django_countries was removed from Portal. We planned to use django-cities-light for all our use cases.